packages/@aws-cdk/aws-rds/test/test.cluster.ts

```diff
@@ -233,7 +290,21 @@ export = {
     }));
 
     test.done();
-  }
+  },
+
+  'empty instance parameter group is not valid'(test: Test) {
+    const stack = testStack();
+    new InstanceParameterGroup(stack, 'Params', {
+      family: 'hello',
+      description: 'desc',
+      parameters: {},
```

Comment:
I don't think we should allow you to create an invalid instance in the first place.